tests/test_kinematicmodel.py

```diff
@@ -2,16 +2,22 @@
 
 import math
 import numpy as np
+import pytest
 
 
-def test_compute_actuators_motion():
+@pytest.fixture
+def get_model():
     alpha = np.array([0, math.pi / 2, math.pi * 3 / 4, math.pi])
     l = np.array([1.0] * 4)
     b = np.array([0.0] * 4)
     r = np.array([0.1] * 4)
     k_beta = 1.0
 
-    km = KinematicModel(alpha, l, b, r, k_beta)
+    return KinematicModel(alpha, l, b, r, k_beta)
+
+
+def test_compute_actuators_motion():
+    km = get_model()
```

Comment:
I'm confused. Is `get_model` supposed to be a fixture, or not?

Reply:
It is. I fixed this on my fork on my `tests` branch. Should I push that stuff onto this PR?